cli/cmd/alert_channel.go

```diff
@@ -0,0 +1,220 @@
+	var out [][]string
+	for _, cadata := range integrations {
+		out = append(out, []string{
+			cadata.IntgGuid,
+			cadata.Name,
+			cadata.Type,
+			cadata.Status(),
+			cadata.StateString(),
+		})
+	}
+	return out
+}
+
+func alertChannelList(_ *cobra.Command, _ []string) error {
+	alertChannels, err := cli.LwApi.V2.AlertChannels.List()
+
+	if err != nil {
+		return errors.Wrap(err, "unable to get alert channels")
+	}
+
+	if len(alertChannels.Data) == 0 {
+		cli.OutputHuman("No alert channels found.\n")
+		return nil
+	}
+
+	if cli.JSONOutput() {
+		return cli.OutputJSON(alertChannels.Data)
+	}
+
+	cli.OutputHuman(
+		renderSimpleTable(
+			[]string{"alert channel GUID", "Name", "Type", "Status", "State"},
+			alertChannelsToTable(alertChannels.Data),
+		),
+	)
+	return nil
+}
+
+func alertChannelShow(_ *cobra.Command, args []string) error {
+	var (
+		alertChannel api.AlertChannelResponse
+		out          [][]string
+	)
+	cli.StartProgress(" Fetching alert channel...")
+	err := cli.LwApi.V2.AlertChannels.Get(args[0], &alertChannel)
+	cli.StopProgress()
+	if err != nil {
+		return errors.Wrap(err, "unable to retrieve alert channel")
+	}
+
+	out = append(out, []string{alertChannel.Data.IntgGuid,
+		alertChannel.Data.Name,
+		alertChannel.Data.Type,
+		alertChannel.Data.Status(),
+		alertChannel.Data.StateString()})
+
+	cli.OutputHuman(renderSimpleTable([]string{"Alert Channel GUID", "Name", "Type", "Status", "State"}, out))
+	cli.OutputHuman("\n")
+	cli.OutputHuman(buildDetailsTable(alertChannel.Data))
+	return nil
+}
+
+func alertChannelCreate(_ *cobra.Command, _ []string) error {
+	if !cli.InteractiveMode() {
+		return errors.New("interactive mode is disabled")
+	}
+
+	err := promptCreateAlertChannel()
+	if err != nil {
+		return errors.Wrap(err, "unable to create integration")
+	}
+
+	cli.OutputHuman("The integration was created.\n")
+	return nil
+}
+
+func alertChannelDelete(_ *cobra.Command, args []string) error {
+	cli.StartProgress(" Deleting alert channel...")
+	err := cli.LwApi.V2.AlertChannels.Delete(args[0])
+	cli.StopProgress()
+	if err != nil {
+		return errors.Wrap(err, "unable to delete alert channel")
+	}
+	cli.OutputHuman("The alert channel %s was deleted.\n", args[0])
+	return nil
+}
+
+func promptCreateAlertChannel() error {
+	var (
+		integration = ""
+		prompt      = &survey.Select{
+			Message: "Choose an integration type to create: ",
+			Options: []string{
+				"Slack Alert Channel",
+				"Email Alert Channel",
+				"Amazon S3 Alert Channel",
+				"Cisco Webex Alert Channel",
+				"Datadog Alert Channel",
+				"GCP PubSub Alert Channel",
+				"Microsoft Teams Alert Channel",
+				"New Relic Insights Alert Channel",
+				"Webhook Alert Channel",
+				"VictorOps Alert Channel",
+				"Splunk Alert Channel",
+				"QRadar Alert Channel",
+				"Service Now Alert Channel",
+				"PagerDuty Alert Channel",
+				"Amazon CloudWatch Alert Channel",
+				"Jira Cloud Alert Channel",
+				"Jira Server Alert Channel",
```

Comment:
Should we drop the ` Alert Channel` string from all of these options? it is now clear that the user is creating an alert channel.